front/src/LeafletMap.js

```diff
@@ -53,24 +53,46 @@ class LeafletMap extends Component {
 
   updateMarkers() {
     const { addresses } = this.props
-    const markerToRemove = this.markers.getLayers().filter(marker => !addresses.find(address => address === marker))
+    const markerToRemove = this.getMarkerToRemove()
     markerToRemove.map(marker => this.markers.removeLayer(marker))
 
     addresses.map(address => {
       const marker = this.getMarker(address)
       if (!marker) {
         const newMarker = this.createMarker(address)
         this.markers.addLayer(newMarker)
-      } else {
+      } else if (!this.isSameLatLng(marker, address)) {
         const latLng = new L.latLng(address.coords.latitude, address.coords.longitude)
         marker.setLatLng(latLng)
         marker.options.address = address.address
       }
     })
   }
 
+  isSameLatLng(marker, address) {
```

Comment:
Pas d'utilisation de this, donc pas utile d'en faire une méthode de classe.
Ça pourrait être une simple fonction indépendante.